dependent/academic-psychiatry.csl

```diff
@@ -2,17 +2,15 @@
 <style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
   <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer -->
   <info>
-    <title>Modern Rheumatology</title>
-    <title-short>Mod Rheumatol</title-short>
-    <id>http://www.zotero.org/styles/modern-rheumatology</id>
-    <link href="http://www.zotero.org/styles/modern-rheumatology" rel="self"/>
+    <title>Academic Psychiatry</title>
+    <id>http://www.zotero.org/styles/academic-psychiatry</id>
+    <link href="http://www.zotero.org/styles/academic-psychiatry" rel="self"/>
```

Comment:
Best documentation on this I've found is http://stackoverflow.com/questions/24753281/git-thinks-ive-renamed-a-file-rather-than-deleting-one-and-creating-another. So it looks like the only option would be to separate the deletion and creation of styles into separate commits, which is unwieldy.